setup.py

```diff
@@ -84,7 +84,7 @@ def _copy_script(src, dest):
     packages=find_packages(),
     install_requires=[
         'pyserial',
-        'ipaddress',
+        'ipaddress;python_version<"3.3"',
```

Comment:
This seems inconsistent with the PR title. If ">= 3.4", then the inverse should be "< 3.3" or "< 3.4"?

Reply:
My apologies, the pull request title is in error.

It is correctly `<3.3` - see [here](https://github.com/phihag/ipaddress) for an authoritative reference.